config.yaml

```diff
@@ -405,8 +405,8 @@ languages:
           icon: fas fa-exclamation-circle
           text: 보안 위반 시 조치를 취합니다. Falco API를 이용해 악의적인 행동에 대해 강력한 대응 애플리케이션을 구축합니다.
       featuredvideo:
-        title: 인기있는 영상
-        subtitle: "더 많은 비디오보기"
+        title: 인기 있는 영상
```

Comment:
Thanks for fixing this... 

I did my best when I wrote it 😃

Reply:
Good Job! Thanks for getting this done 😄